docs/docs/xmus.md

```diff
@@ -0,0 +1,38 @@
+xemu allows inserting XMUs (Xbox Memory Units) to controller hubs.
+
+This is done by using a file as the storage medium. Any file bigger than 1 MiB in size should work.
+
+
+## Creating the XMU
```

Comment:
* Maybe this title can be more clearly stated as "Creating the XMU Device" or
* This should probably come after the section about "Creating the XMU Image"